android/sdl_android/src/main/java/com/smartdevicelink/managers/video/resolution/VideoStreamingRange.java

```diff
@@ -0,0 +1,61 @@
+package com.smartdevicelink.managers.video.resolution;
+
+public class VideoStreamingRange {
+    private Resolution minSupportedResolution;
+    private Resolution maxSupportedResolution;
+    private Double maxScreenDiagonal;
+    private AspectRatio aspectRatio;
+
+    public VideoStreamingRange(Resolution minSupportedResolution, Resolution maxSupportedResolution, Double maxScreenDiagonal, AspectRatio aspectRatio) {
+        this.minSupportedResolution = minSupportedResolution;
```

Comment:
Per the proposal, VideoStreamingRange should be defined as:
"...
public class VideoStreamingRange {
private Resolution minResolution;
private Resolution maxResolution;
private Double minAspectRatio;
private Double maxAspectRatio;
private Double minScreenDiagonal;
}
..."

Do you need to change your variable definitions to match with the proposal?